appbox_installer.sh

```diff
@@ -1092,6 +1092,67 @@ EOF
 #     configure_nginx 'deemixrr' '5555'
 # }
 
+# Based on: https://github.com/mynttt/UpdateTool/issues/70
+setup_updatetool() {
+    s6-svc -d /run/s6/services/updatetool || true
+    apt install -y libcurl4-openssl-dev bzip2 default-jre
+    cd /home/appbox/appbox_installer
+    mkdir -p /home/appbox/appbox_installer/UpdateTool
+    cd /home/appbox/appbox_installer/UpdateTool
+    curl -L -O $( curl -s https://api.github.com/repos/mynttt/UpdateTool/releases/latest | grep UpdateTool | grep browser_download_url | head -1 | cut -d \" -f 4 )
+    chown -R appbox:appbox /home/appbox/appbox_installer/UpdateTool    
+    chmod +x /home/appbox/appbox_installer/UpdateTool/UpdateTool-1.6.3.jar  
+    echo "#!/bin/bash
+    cd /home/appbox/appbox_installer/UpdateTool && ./runner.sh" > /home/appbox/appbox_installer/UpdateTool/run-runner.sh
+    chmod +x /home/appbox/appbox_installer/UpdateTool/run-runner.sh
+    echo "#!/bin/bash
+
+# Must point to a java installation >= java 11
+export JAVA=\"/usr/bin/java\"
+
+# Points to the tool jar file
+export TOOL_JAR=\"/home/appbox/appbox_installer/UpdateTool/UpdateTool-1.6.3.jar\"
+
+# Change to 512m or 1g if tool crashes with out of memory exception
+export JVM_MAX_HEAP=\"-Xmx256m\"
+
+## Run batch processing every hours
+export RUN_EVERY_N_HOURS=\"12\"
+
+# Set environment variables (must be exported for each var): See https://github.com/mynttt/UpdateTool#environment-variables-guide
+
+## REQUIRED: Data dir
+PLEX_DATA_DIR=\"/APPBOX_DATA/apps/plex.${HOSTNAME}/config/Library/Application Support/Plex Media Server/\"
```

Comment:
Here's the issue causing #55, you cannot just use $HOSTNAME, you need to remove the first subdomain (normally Ubuntu).